#!/usr/bin/bash

def raise_exception_msg(message=""):
    print(message)
